testing/scenario_app/android/README.md

```diff
@@ -0,0 +1,42 @@
+# Scenario App: Android Tests
+
+As mentioned in the [top-level README](../README.md), this directory contains
+the Android-specific native code and tests for the [scenario app](../lib). To
+run the tests, you will need to build the engine with the appropriate
+configuration.
+
+For example, `android_debug_unopt` or `android_debug_unopt_arm64` was built,
+run:
+
+```sh
+# From the root of the engine repository
+$ ./testing/run_android_tests.sh android_debug_unopt
+
+# Or, for arm64
+$ ./testing/run_android_tests.sh android_debug_unopt_arm64
+```
+
+## CI Configuration
+
+See [`ci/builders/linux_android_emulator.json`](../../../ci/builders/linux_android_emulator.json)
+, and grep for `run_android_tests.sh`.
+
+The following matrix of configurations is tested on the CI:
+
+| API Version | Graphics Backend    | Skia Gold                              | Rationale                                                  |
+| ----------- | ------------------- | -------------------------------------- | ---------------------------------------------------------- |
+| 28          | Skia                | [Link][skia-gold-skia-28]              | Older Android devices (without `ImageReader`) on Skia.     |
```

Comment:
nit: give these links apropos names instead of "Link".

Reply:
Done.